lib/sufia/import/version_builder.rb

```diff
@@ -3,19 +3,16 @@ module Sufia::Import
   class VersionBuilder
     attr_reader :file_set
 
-    # @param file_set FileSet that will be modifed to include versions
-    def initialize(file_set)
-      @file_set = file_set
-    end
-
     # build versions based on the input
     #
+    # @param file_set FileSet that will be modifed to include versions
     # @param Array[hash] generic_file_versions, each with the keys below
-    # @option :uri Link to content in Sufia 6 repository
-    # @option :label Version label
-    # @option :created date the version was created
+    #   @option :uri Link to content in Sufia 6 repository
+    #   @option :label Version label
+    #   @option :created date the version was created
     #
-    def build(generic_file_versions)
+    def build(file_set, generic_file_versions)
+      @file_set = file_set
```

Comment:
@hackmastera this one is a bit more complex, but I still think passing it would make it more obvious later where the information is coming from.